requirements/tox-static-analysis-requirements.txt

```diff
@@ -0,0 +1,191 @@
+#
```

Comment:
This file is the requirements file for static code analysis execution 

pip-compile requirements/tox-static-code-analysis.in -o requirements/tox-static-analysis-requirements.txt --resolver=backtracking